dbt/include/trino/macros/materializations/snapshot.sql

```diff
@@ -1,6 +1,34 @@
+{% macro trino__snapshot_hash_arguments(args) -%}
+  lower(to_hex(md5(to_utf8(concat({%- for arg in args -%}
```

Comment:
Any particular reason for using `lower` here (readability)? I suppose it's redundant.

Reply:
It conforms to the md5 dbt function as in dbt-utils.

Reply:
The function signature of md5 is md5(binary) → varbinary [Trino docs](https://trino.io/docs/current/functions/binary.html#md5)

The `to_hex` need to convert it back to VARCHAR. The output of `to_hex` is in uppercase.

When using md5 in other dbt adapters the output is in lowercase.